build/request.js

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
 /**
  * @module request
  */
-var Promise, errors, estimate, prepareOptions, progress, request, requestAsync, settings, stream, token, url, utils, _;
+var Promise, errors, estimate, estimateRequestStream, prepareOptions, progress, request, requestAsync, rindle, settings, stream, token, url, utils, _;
```

Comment:
Line is too long.<br>'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.